### PR TITLE
feat: add domain settings for project dashboard

### PR DESCRIPTION
**Pull request by 4nalog**
# TL;DR
This PR adds support for showing project attributes and project domain attributes in the Project Dashboard for flyteconsole.

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description

Currently, there is no way for users to directly tell project attributes, project domain attributes. This PR adds support for showing project attributes and project domain attributes in the Project Dashboard for flyteconsole.

Note: We currently merge the project attributes with the project domain attributes in a single view. In future and as per designs, we will indicate separately global and project level domain settings so that users can see what changed

## Tracking Issue

[https://github.com/flyteorg/flyteconsole/issues/382](https://github.com/flyteorg/flyteconsole/issues/382)

## Follow-up issue
_https://github.com/flyteorg/flyteconsole/issues/382_

### CODE REVIEW

File: packages/console/src/components/Project/ProjectDashboard.tsx [hunk 1]
```diff
@@ -166,7 +186,7 @@ export const ProjectDashboard: React.FC<ProjectDashboardProps> = ({
       );
       return projectDomainAtributes;
     },
-    staleTime: Infinity,
+    enabled: !!projectAttributesQuery.data,
```
Comment (4nalog) [inline packages/console/src/components/Project/ProjectDashboard.tsx:189]: The behaviour when the project settings error out is not clear. So we fetch projectDomainAttributes only once we have the upper level query.

**Review by 4nalog — COMMENTED** [review]
added some explanatory comments

**Review by jsonporter — DISMISSED** [review]

File: packages/console/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@flyteorg/console",
-  "version": "0.0.8",
+  "version": "0.0.9",
```
Comment (ursucarina) [inline packages/console/package.json:3]: :raised_hands:

**Review by jsonporter — APPROVED** [review]
